source/style/content.css

```diff
@@ -33,6 +33,19 @@ body {
 	border: 0 !important;
 }
 
+/* Highlight Mention in stream */
+[data-mentioned-user-id] {
+    position: relative;
+    padding: 2px 5px 4px 5px;
+}
+
+.tweet.mention4me {
```

Comment:
Can you change the name for `.refined-twitter-mention`? This way we male it clear that this was added by the extension and avoid conflicts :)